src/libcollections/str.rs

```diff
@@ -1365,22 +1610,43 @@ impl str {
         core_str::StrExt::trim_right_matches(self, pat)
     }
 
-    /// Parses `self` into the specified type.
+    /// Parses this string slice into another type.
+    ///
+    /// Because `parse()` is so general, it can cause problems with type
+    /// inference. As such, `parse()` is one of the few times you'll see
+    /// the syntax affectionately known as the 'turbofish': `::<>`. This
```

Comment:
I haven't been using curly quotes in API docs since we have never really used them. I'd like to convert them all someday.